langchain/vectorstores/weaviate.py

```diff
@@ -201,6 +201,8 @@ def similarity_search_by_text(
         query_obj = self._client.query.get(self._index_name, self._query_attrs)
         if kwargs.get("where_filter"):
             query_obj = query_obj.with_where(kwargs.get("where_filter"))
+        if kwargs.get("additional"):
```

Comment:
does this apply to other search methods as well? (like similarity_search_by_vector)

Reply:
Yes, we can use the same mechanism in the function _similarity_search_by_vector_. They have different values that you can use in the additional field. But that is specified in the Weaviate documentation. Not sure if we should replicate that here as well.

I would not use it in the functions similarity_search_with_score and max_marginal_relevance_search_by_vector. They allready use the mechanism with the additional field. They already return a score.

Do you want me to add it to the similarity_search_by_vector method as well?

Reply:
that'd be great!